vocab_files/attribute_concepts/is-confident.ttl

```diff
@@ -11,8 +11,8 @@ Ecological Field Monitoring Protocols Manual: Standardising environmental monito
 decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
     skos:definition "The level of confidence while identifying a species during an ecological survey." ;
```

Comment:
Probably worth changing the definition as well.

Reply:
How about `Whether the surveyor is confident.`?

Reply:
Do you mean like this?

```suggestion
    skos:definition "Whether the surveyor is confident in identifying the species during an ecological survey." ;
```

Reply:
Yes, can I commit the suggestion?

Reply:
Yeah sure!